internal/app/machined/pkg/controllers/block/internal/kobject/kobject.go

```diff
@@ -77,7 +77,7 @@ func (w *Watcher) Run(logger *zap.Logger) <-chan *Event {
 		for {
 			ev, err := w.cli.Receive()
 			if err != nil {
-				if err.Error() == "use of closed file" { // unfortunately not an exported error, just errors.New()
+				if err.Error() != "use of closed file" { // unfortunately not an exported error, just errors.New()
```

Comment:
bug fix?

Reply:
yes, I fixed the error mapping, but the condition was reversed :facepalm: